index-graphclust.md

```diff
@@ -5,46 +5,61 @@ layout: subsite-galaxy
 
 <br/>
 
-Welcome to **Galaxy GraphClust** -- a webserver to process and analyse structural clustering of RNA secondary structures.
+Welcome to **GraphClust2** Galaxy server for clustering of RNAs according to sequence and secondary structures similarities.
```

Comment:
There as never a GraphClust 1 in Galaxy isn't it?

Reply:
Yes, only GC2

Reply:
Aha now I get it, to keep the naming renamed Galaxy-GraphClusts to GraphClust2.